generators/languages/index.js

```diff
@@ -0,0 +1,126 @@
+         *              // Do all your stuff here
+         *          },
+         *          myAnotherCustomInitPhaseStep(){
+         *              // Do all your stuff here
+         *          }
+         *      };
+         * ```
+         *
+         * 3. Partially override a phase, this is when the blueprint gets the phase from JHipster and customizes it.
+         * ```
+         *      const phaseFromJHipster = super._initializing();
+         *      const myCustomPhaseSteps = {
+         *          displayLogo() {
+         *              // override the displayLogo method from the _initializing phase of JHipster
+         *          },
+         *          myCustomInitPhaseStep() {
+         *              // Do all your stuff here
+         *          },
+         *      }
+         *      return Object.assign(phaseFromJHipster, myCustomPhaseSteps);
+         * ```
+         */
+        // Here we are not overriding this phase and hence its being handled by JHipster
+        return super._initializing();
+    }
+
+    get prompting() {
+        // Here we are not overriding this phase and hence its being handled by JHipster
+        return super._prompting();
+    }
+
+    get configuring() {
+        // Here we are not overriding this phase and hence its being handled by JHipster
+        return super._configuring();
+    }
+
+    get default() {
+        const phaseFromJHipster = super._default();
+        const myCustomPhaseSteps = {
+            getSharedConfigOptions() {
+                if (this.configOptions.applicationType) {
+                    this.applicationType = this.configOptions.applicationType;
+                }
+                if (this.configOptions.baseName) {
+                    this.baseName = this.configOptions.baseName;
+                }
+                if (this.configOptions.websocket !== undefined) {
+                    this.websocket = this.configOptions.websocket;
+                }
+                if (this.configOptions.databaseType) {
+                    this.databaseType = this.configOptions.databaseType;
+                }
+                if (this.configOptions.searchEngine !== undefined) {
+                    this.searchEngine = this.configOptions.searchEngine;
+                }
+                if (this.configOptions.messageBroker !== undefined) {
+                    this.messageBroker = this.configOptions.messageBroker;
+                }
+                if (this.configOptions.enableTranslation) {
+                    this.enableTranslation = this.configOptions.enableTranslation;
+                }
+                if (this.configOptions.nativeLanguage) {
+                    this.nativeLanguage = this.configOptions.nativeLanguage;
+                }
+                if (this.configOptions.skipClient) {
+                    this.skipClient = this.configOptions.skipClient;
+                }
+
+                // disable languages
+                this.skipServer = true;
```

Comment:
Why set skipServer true?

Reply:
This task is just for client, so I have to skipServer to prevent message_xxx.properties generation.
Or if we want to deal with i18n server, funtion getSharedConfigOptions  should be removed.

Reply:
Yeh, I know. But I don't know another way to make this task just 'i18n client'.
This is the first time I collaborate for an open source. So please patient with me :)
I've had some interviews this week. I'm going to try better at next week.

Reply:
Not a problem! Feel you free to contribute when you want, for me it's already an excellent job! Thanks a lot!

Reply:
@amanganiello90 When I use 'npm run test', there's no error detail. But when I request to pull code to this branch, the detail will appear and show me the reason. How are you testing in your local?

Reply:
I have commented sonar task, I don't see build error.